Makefile

```diff
@@ -76,12 +76,15 @@ $(OUTPUT_DIRNAME)/$(DOC_FILENAME).html: $(DOC_FILES) $(FIGURE_FILES)
 	ls -sh $(shell readlink -f $@)
 endif
 
-validate-examples:
+validate-examples: schema/fs.go
```

Comment:
> then i'll add validate-examples as a travis target.

That will double those tests, since `make test`, which we already call, also runs the example tests.  Other `make test` tests depend on `fs.go` as well, so I think you can leave Travis alone and add the `fs.go` dependency to `test`.